tv-shows/src/components/features/shows/TopRatedShowsContainer.tsx/TopRatedShowsContainer.tsx

```diff
@@ -2,14 +2,15 @@
 
 import useSWR from 'swr';
 import { getManyTopRatedShows } from '../../../../fetchers/show';
+import { ErrorMessage } from '../../../shared/ErrorMessage/ErrorMessage';
 import { Loader } from '../../../shared/Loader/Loader';
 import { ShowsList } from '../ShowsList/ShowsList';
 
 export function TopRatedShowsContainer() {
 	const { data, error, isLoading } = useSWR('/api/shows/top-rated', getManyTopRatedShows);
```

Comment:
Should the key be from `swrKeys.topRated`?

Reply:
Fixed in https://github.com/infinum-academy/js-react-academy-2024-danipavic/pull/5/files#diff-1cf2aa9ce5fde505db539b1ff31c268fb2f06a6360faee95571840243eddd865